### PR TITLE
Add RAW capabilites

### PR DESCRIPTION
**Pull request by mathialo**
Add a RAW upload queue and allow handle functions to return rows.

Since we need to know the db/table (and those are not attributes in the
Row class from the SDK), I think we need to define our own types. To
avoid possible confusion, and in case we need to do similar things for
other resource types, I added a types module, which contains type
aliases for those types where we just use the SDK types directly. That
way, users can always import from our type module and be safe.

### CODE REVIEW

File: example.py [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
 
-from cognite.client.data_classes import Event
+from cognite.client.data_classes import Row
 
 from cognite.extractorutils.rest import RestExtractor
+from cognite.extractorutils.rest.types import Event, RawRow
 
 
 @dataclass
 class RawEvent:
```
Comment (einarmo) [inline example.py:12]: I'm not quite sure why we need to do this?

**Review by einarmo — APPROVED** [review]
Looks good, but why do we need the whole "RawEvent" thing?

File: example.py (hunk 1 above)
Comment (mathialo) [reply example.py:12]: Couple of reasons (notice this is the example file)
 * To show a slightly more realistic example where the source data model is not the destination data model directly
 * Easy workaround for casing changes (we should handle that in the library at some point)
 * `dacite`, which is the best turn-a-dict-to-a-class-instance library I know of, can only serialize `@dataclass`es, which the `Event` class from the SDK is not

Comment (einarmo) [reply example.py:12]: I see. Sounds reasonable!

**Pull request merged**